src/vopr.zig

```diff
@@ -138,7 +138,7 @@ fn build_simulator(
     mode: std.builtin.Mode,
 ) void {
     const mode_str = switch (mode) {
-        .Debug => "-Ddebug",
+        .Debug => "--",
```

Comment:
What about using `-Drelease-safe=false` and `-Drelease-safe=true`? `--` works but might be surprising if we add other arguments in the future.